Add follow up survey draft sent to IRB (#1)
fhdsl/ITN_Surveys: itn_follow_Up/survey.qmd

    "Case studies"                                                                                                                                 = "case_studies",
    "None of the above"                                                                                                                            = "none",
    "Other"                                                                                                                                        = "other"
  )
)

sd_question(
  type  = "text",
  id    = "modalities_other",
  label = '*Optional* If you answered "Other" to the previous question, please elaborate:'
)

sd_question(
  type   = 'matrix',
  id     = 'ai_daily_work',
  label  = 'What are your thoughts about the impact of AI on your <b>daily work</b>?',
  row    = c(
    "I fully trust the output of AI tools"                                                              = "trust_output",
    "Using AI tools effectively can be tricky"                                                          = "effectively_difficult",

>     "Using AI tools ethically is challenging"                                                           = "ethically_difficult",

Reviewer: maybe we can split into a code question and non-code question?

Reviewer: could include an NA option and split out the last question part into two pieces

    'Read the <a href="https://www.itcrtraining.org/resources/blog" target="_blank">ITN Blog</a>'                                            = "read_blog",
    'Used the <a href="https://www.itcrtraining.org/resources/computing-and-data" target="_blank">ITN computing resource table</a>'          = "browse_computing_table",
    'Attended an <a href="https://www.itcrtraining.org/open" target="_blank">ITCR OPEN meeting</a>'                                          = "open_meeting",
    'Attended or presented at an <a href="https://sites.google.com/view/itcrtraineesymposium" target="_blank">ITCR Trainee Symposium</a>'    = "trainee_symposium",
    "Other"                                                                                                                                  = "other"
  )
)

sd_question(
  type  = 'text',
  id    = 'engaged_with_other',
  label = '*Optional* If you answered "Other" to the previous question, please elaborate:'
)

sd_question(
  type   = 'matrix',
  id     = 'observed_changes',
  label  = "Which of the following statements accurately describes changes you've observed after interacting with ITN resources?",
  row    = c(

>     "I have a better understanding of my work and the tools I use"      = "better_understanding",

Reviewer: ```suggestion
    "I better understand my work and the tools I use"      = "better_understanding",
```

>     "I am able to do my work more efficiently"                          = "more_efficient",

Reviewer: ```suggestion
    "I can work more efficiently"                          = "more_efficient",
```

>     "My ability to evaluate different scientific questions is enhanced" = "evaluation_of_qs",

Reviewer: ```suggestion
    "I can now ask different scientific questions" = "evaluation_of_qs",
```

>     "I have an easier time doing some of my work"                       = "easier",

Reviewer: ```suggestion
    "Some of my work is now easier"                       = "easier",
```

>     "My work is more transparent and I can better track my steps"       = "more_transparent",

Reviewer: ```suggestion
    "I can better track the steps of my work"       = "more_transparent",
```

Author: Going to take a space off the front of this one before I commit it

Reviewer: Good catch! I can edit it for you.

Reviewer: fixed!

  option = c(
    "Reproducibility practices (e.g., code documenting, code review, automation, etc.)" = "reproducibility",
    "Communication (e.g., LaTex/Overleaf use, clear documentation, etc.)" = "communication",
    "Efficiency (e.g., use of AI or automation, computing resources)" = "efficiency",
    "Leadership skills" = "leadership",
    "Ethical considerations" = "ethics",
    "Data management and sharing" = "data_handling",
    "Knowledge or use of bioinformatic and computing platforms and tools (e.g., Galaxy, GenePattern, Xena, etc.)" = "tools",
    "I recently used ITN resources, so there likely hasn't been enough time to witness change" = "recent_use",
    "I did not observe positive changes" = "no_observed",
    "Other" = "other"
  )
)

sd_question(
  type = 'text',
  id = 'categories_observed_changes_other',
  label = '*Optional* If you answered "Other" to the previous question, please elaborate:'
)

sd_question(
  type = 'matrix',
  id = 'return_recommend_freq',
  label = "Select which statement is most true for your experience after engaging with ITN content.",
  row = c(

>     "I have returned to the content to help with my work" = "returned",

Reviewer: ```suggestion
    "I have returned to the content" = "returned",
```

>     "I have recommended the content to my mentor, colleagues, lab mates, or mentees" = "recommended"

Reviewer: ```suggestion
    "I have recommended the content to others" = "recommended"
```

Author: Really like this suggestion after test taking the survey!

  ),
  option = c(
    "Never" = "never",

>     "I haven't/NA" = "havenot",

Reviewer: what is the difference in the thinking again about never vs haven't?

Author: Not sure -- can simplify and remove one

>     "I haven't, but plan to" = "plan_to",

Reviewer: ```suggestion
    "Plan to" = "plan_to",
```

    "Once or twice" = "a_couple",

>     "Occasionally" = "occasionally",

Reviewer: ```suggestion

```

Reviewer: maybe we can drop this one?

Author: Agreed.

>     "Frequently" = "frequently"

Reviewer: ```suggestion
    "Often" = "often"
```
